modules.d/95resume/module-setup.sh

```diff
@@ -13,7 +13,7 @@ check() {
     # Only support resume if hibernation is currently on
     # and no swap is mounted on a net device
     [[ $hostonly ]] || [[ $mount_needs ]] && {
-        swap_on_netdevice || [[ -f /sys/power/resume && "$(cat /sys/power/resume)" == "0:0" ]] && return 255
+        swap_on_netdevice || [[ -f /sys/power/resume && "$(< /sys/power/resume)" == "0:0" ]] || grep -rq '^\|[[:space:]]resume=' /proc/cmdline /etc/cmdline /etc/cmdline.d "${initdir}/etc/cmdline" "${initdir}/etc/cmdline.d" /etc/kernel/cmdline /usr/lib/kernel/cmdline 2> /dev/null && return 255
```

Comment:
afaict that only covers some paths, and we process a superset of those here, so I don't think it applies

Reply:
AIUI $initdir is populated by modules, so it could be.

dracut --kernel-cmdline resolves to `printf "%s\n" "$kernel_cmdline" >> "${initdir}/etc/cmdline.d/01-default.conf"` on EFI and to `echo -n "$kernel_cmdline" > "$uefi_outdir/cmdline.txt"` so, per the first question, yes.

No. Unless you're using sd-boot, in which case it ends up in the kernel cmdline, so yes. Don't ask me why that's the only bootloader we special-case, I gave up trying to understand the mental gymnastics upthread and I don't care anymore.

Reply:
oh i forgor 💀 this is in check(); removed the `$initdir`s